lib/matplotlib/legend.py

```diff
@@ -130,29 +131,31 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
     artists.  This option can be quite slow for plots with large amounts of
     data; your plotting speed may benefit from providing a specific location.
 
+    For back-compatibility, ``'center right'`` (but no other location) can also
```

Comment:
Simpler:
~~~
For backward-compatibility, ``'right'`` is equivalent to  ``'center right'``;
and there are numeric codes for all locations as well. 
~~~